docs/maintenance/Pull_Requests.mdx

```diff
@@ -111,6 +108,22 @@ For preliminary reviews, be clear with what scale you're reviewing at: _"Reviewi
 For repeat reviews, be clear when it's something you missed earlier and/or there's new information.
 Don't apologize if the missed information was only made clear because of requested changes - this is part of the review process!
 
+### Approvals
+
+Upon completing your review, if the build is passing and you have no blockers, approve it on GitHub.
+Then:
+
+- If there isn't a `1 approval` label or existing approval, add the `1 approval` label
```

Comment:
Oh, I wonder if we should add a contingency for urgent things, like security reports or broken `main` builds...

Reply:
3abac143f